<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Malas-MBP.attlocal.net" name="USPS Reports" tests="4" failures="2" timestamp="27 Mar 2023 07:06:07 GMT" time="75.349" errors="0">
  <testcase name="SetFunEnv" time="0.071" classname="CommonUtility.SetEnvStaging"/>
  <testcase name="uspsCalTest1" time="18.244" classname="qa.vector.UspsCalculatorTest"/>
  <testcase name="uspsCalTest2" time="17.309" classname="qa.vector.UspsCalculatorTest2">
    <failure type="java.lang.AssertionError" message="Price is higher than $80 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Price is higher than $80 expected [true] but found [false]
	at qa.vector.UspsCalculatorTest2.uspsCalTest2(UspsCalculatorTest2.java:125)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- uspsCalTest2 -->
  <testcase name="uspsCalTest3" time="17.165" classname="qa.vector.UspsCalculatorTest3">
    <failure type="java.lang.AssertionError" message="Price is higher than $80 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Price is higher than $80 expected [true] but found [false]
	at qa.vector.UspsCalculatorTest3.uspsCalTest3(UspsCalculatorTest3.java:129)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- uspsCalTest3 -->
</testsuite> <!-- USPS Reports -->
